### ISSUE TITLE
board support kubernetes node name which is not ip address.

### ISSUE THREAD

**Issue by inspuradmin** [inspursoft/board#926]
In GitLab by @tmon546596046 on Nov 25, 2020, 02:40

helm is using kubernetes node ip, but currently node ip is nodename...
```
// adapt model node from k8s node
func FromK8sNode(node *v1.Node) *model.Node {

	return &model.Node{
		ObjectMeta:    FromK8sObjectMeta(node.ObjectMeta),
		NodeIP:        node.ObjectMeta.Name,
		Unschedulable: node.Spec.Unschedulable,
		Taints:        FromK8sNodeTaints(node.Spec.Taints),
		Status:        FromK8sNodeStatus(node.Status),
	}
}
```

the NodeIP is setted to nodename.

**Comment by inspuradmin** [comment]
In GitLab by @tmon546596046 on Nov 25, 2020, 02:40

we should set the nodeip for valid ip address.

**Comment by tmon546596046** [comment]
fixed #1711
